src/custom/theme/cowSwapTheme.tsx

```diff
@@ -25,45 +24,50 @@ export function colors(darkMode: boolean): Colors {
     ...colorsBaseTheme(darkMode),
 
     // ****** base ******
-    white: '#FFFFFF',
-    black: '#000000',
+    white: darkMode ? '#c5daef' : '#ffffff',
+    black: darkMode ? '#021E34' : '#000000',
 
     // ****** text ******
-    text1: darkMode ? '#000000' : '#000000',
-    text2: darkMode ? '#000000' : '#000000',
-    text3: darkMode ? '#000000' : '#000000',
-    text4: darkMode ? '#000000b8' : '#000000b8',
+    text1: darkMode ? '#c5daef' : '#000000',
+    text2: darkMode ? '#021E34' : '#000000',
```

Comment:
why not re-enable all the `text2` code you commented out and make it work with a darkmode toggle here? why target specifically with CSS? all the commented color prop code was for `text2` no?

Reply:
The reason is that, in comparison with the UNI front-end, on our end it's not a 1-on-1 translation in terms of switching out colors: On several occasions we use a different color contrast and thus our existing variables don't match that scenario (e.g. resulting in a dark text on a dark background).